.clang-format

```diff
@@ -10,6 +10,7 @@ AlignTrailingComments: true
 AllowAllParametersOfDeclarationOnNextLine: true
 AllowShortBlocksOnASingleLine: Never
 AllowShortCaseLabelsOnASingleLine: true
+AllowShortEnumsOnASingleLine: false
```

Comment:
Neither `true` nor `false` seems to match the current style where we often have a few enumeration values listed per lines (in a table like structure) but this is probably the least bad option.